test/plugin-cloud-storage/.env.emulated

```diff
@@ -1,6 +1,5 @@
 # Sample creds for working locally with docker-compose
 
-MONGODB_URI=mongodb://localhost/payload-plugin-cloud-storage
```

Comment:
Don't know why we had this - this won't work anymore. It should use the default URL or whatever env variable the CI action sets